### PR TITLE
[Feature][zeta]Support exposing monitoring metrics by prometheus exporter protocol

### PR DESCRIPTION
**Pull request by corgy-w**
### Purpose of this pull request

issue: #7539 

ing:       :>
1. Provides complete grafana config template
2. Testing 
3. Continue to add  indicators，See doc for details（comments are welcome）

### How was this patch tested?
e2e

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  4. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  5. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  6. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  7. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @hailin0

File: docs/en/seatunnel-engine/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+
+sidebar_position: 8
+-------------------
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/telemetry.md]:
```suggestion
---
sidebar_position: 8
---
```

File: docs/en/seatunnel-engine/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+# Telemetry
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/telemetry.md:5]: Please update https://github.com/apache/seatunnel/blob/e31c9c2a9a0804fd24f3ad16ab28fa4fd7ecc2bb/docs/sidebars.js#L193

File: docs/zh/seatunnel-engine/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
+---
+
+sidebar_position: 8
+-------------------
```
Comment (Hisoka-X) [inline docs/zh/seatunnel-engine/telemetry.md]: ditto

File: docs/en/seatunnel-engine/telemetry.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+For a guide on how to set up Grafana server go to
+the [Installation](https://grafana.com/docs/grafana/latest/setup-grafana/installation)
+
+#### Monitoring Dashboard
+
+- Add Prometheus DataSource on Grafana.
+  - Import `Seatunnel Cluster` monitoring dashboard by [Dashboard JSON](./telemetry/grafana-dashboard.json) into Grafana.
+
+The effect image of the dashboard
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/telemetry.md]: where is it?

**Comment by Hisoka-X** [comment]
The [grafana-dashboard.json](https://github.com/apache/seatunnel/pull/7564/files#diff-ce66180f2b81eaf7ee32f9a92fb73d2fcbf990b78bb519db8278f18e0104728c) is empty. Please update it.

**Comment by corgy-w** [comment]
> The [grafana-dashboard.json](https://github.com/apache/seatunnel/pull/7564/files#diff-ce66180f2b81eaf7ee32f9a92fb73d2fcbf990b78bb519db8278f18e0104728c) is empty. Please update it.

@Hisoka-X As mentioned above, grafana template will have to wait a little longer, but this week

File: docs/zh/seatunnel-engine/telemetry.md (hunk 3 above)
Comment (corgy-w) [reply docs/zh/seatunnel-engine/telemetry.md]: get

File: docs/en/seatunnel-engine/telemetry.md (hunk 2 above)
Comment (corgy-w) [reply docs/en/seatunnel-engine/telemetry.md:5]: get

File: docs/en/seatunnel-engine/telemetry.md (hunk 1 above)
Comment (corgy-w) [reply docs/en/seatunnel-engine/telemetry.md]: get

**Comment by hailin0** [comment]
good job

**Comment by hailin0** [comment]
Please add monitoring screenshots of grafana

**Comment by corgy-w** [comment]
PTAL @Hisoka-X @hailin0

File: config/seatunnel.yaml [hunk 5]
```diff
@@ -34,3 +34,6 @@ seatunnel:
           namespace: /tmp/seatunnel/checkpoint_snapshot
           storage.type: hdfs
           fs.defaultFS: file:///tmp/ # Ensure that the directory has written permission
+    telemetry:
+      metric:
+        enabled: true
```
Comment (Hisoka-X) [inline config/seatunnel.yaml]: how about turn off by default?

Comment (corgy-w) [reply config/seatunnel.yaml]: get

File: docs/zh/seatunnel-engine/telemetry.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+| jvm_memory_pool_collection_max_bytes       | Gauge   | **池**，包括："PS Eden Space" "PS Old Ge" "PS Survivor Space"                                                       | 给定 JVM 内存池在最后一次回收后的最大字节数               |
+| jvm_memory_pool_collection_init_bytes      | Gauge   | **池**，包括："PS Eden Space" "PS Old Ge" "PS Survivor Space"                                                       | 给定 JVM 内存池在最后一次回收后的初始字节数               |
+| jvm_buffer_pool_used_bytes                 | Gauge   | **池**，包括："direct" "mapped"                                                                                     | 给定 JVM 缓冲池使用的字节数                       |
+| jvm_buffer_pool_capacity_bytes             | Gauge   | **池**，包括："direct" "mapped"                                                                                     | 给定 JVM 缓冲池的字节容量                        |
+| jvm_buffer_pool_used_buffers               | Gauge   | **池**，包括："direct" "mapped"                                                                                     | 给定 JVM 缓冲池使用的缓冲区                       |
+
+### 通过 Prometheus 和 Grafana 进行集群监控
```
Comment (Hisoka-X) [inline docs/zh/seatunnel-engine/telemetry.md]:
```suggestion
## 通过 Prometheus 和 Grafana 进行集群监控
```

**Review by Hisoka-X — COMMENTED** [review]
Only one minior change.

File: docs/zh/seatunnel-engine/telemetry.md (hunk 6 above)
Comment (corgy-w) [reply docs/zh/seatunnel-engine/telemetry.md]: get , revised doc⬇️

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @corgy-w !

**Review by hailin0 — APPROVED** [review]

File: docs/en/seatunnel-engine/telemetry.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+        enabled: true # Whether open metrics export
+```
+
+## Metrics
+
+The [metric text of prometheus](./telemetry/metrics.txt),which get
+from `http:{instanceHost}:5801/hazelcast/rest/instance/metrics`.
+
+The [metric text of openMetrics](./telemetry/openmetrics.txt),which get
+from `http:{instanceHost}:5801/hazelcast/rest/instance/openmetrics`.
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/telemetry.md:28]: The http path looks not right, it missed `//`. Could you create a PR to correct it? @corgy-w

Comment (corgy-w) [reply docs/en/seatunnel-engine/telemetry.md:28]: get

**Pull request merged**